xray/commands/audit/jas/common.go

```diff
@@ -19,8 +19,12 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const (
+	NodeModulesPattern = "**/*node_modules*/**"
```

Comment:
```suggestion
	nodeModulesPattern = "**/*node_modules*/**"
```

Reply:
been used from applicablitymanager.go so has to be public